src/main/java/io/github/queerbric/queerbrics/QueerbricsREIPlugin.java

```diff
@@ -0,0 +1,102 @@
+package io.github.queerbric.queerbrics;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import com.mojang.datafixers.util.Pair;
+
+import org.jetbrains.annotations.NotNull;
+
+import io.github.queerbric.queerbrics.datatypes.item.QueerBrickItem;
+import io.github.queerbric.queerbrics.registry.QueerbricsItems;
+import me.shedaniel.rei.api.EntryRegistry;
+import me.shedaniel.rei.api.EntryStack;
+import me.shedaniel.rei.api.RecipeHelper;
+import me.shedaniel.rei.api.plugins.REIPluginV0;
+import me.shedaniel.rei.plugin.crafting.DefaultCraftingDisplay;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.StringTag;
+import net.minecraft.recipe.Recipe;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+@Environment(EnvType.CLIENT)
+public class QueerbricsREIPlugin implements REIPluginV0 {
+	private static List<Pair<QueerBrickItem, EntryStack>> entries = new ArrayList<Pair<QueerBrickItem, EntryStack>>();
+
+	static {
+		if (FabricLoader.getInstance().isModLoaded("titlescrolls")) {
```

Comment:
This same check is repeated a couple of times in the code; it probably works but one typo in the title scrolls mod id could be hard to debug, so I usually advise to DRY stuff like this (least intrusive way would just be a static string var somewhere).

Reply:
I changed this to a static string for the checks and identifiers that use it.